### PR TITLE
Certified overview laptops

### PR DESCRIPTION
**Pull request by carkod**
## Done

- Certified overview laptops. 

## QA

- Check https://ubuntu-com-9775.demos.haus for the new laptops card

## Issue / Card

Fixes https://github.com/canonical-web-and-design/certification.ubuntu.com/issues/195

### CODE REVIEW

**Comment by bethcollins92** [comment]
There are no numbers after the releases. Also the numbers seem to be exactly the same between laptops and desktops - double checking is it just a coincidence?
image

**Comment by codersquid** [comment]
> Also the numbers seem to be exactly the same between laptops and desktops - double checking is it just a coincidence?

The numbers should not be the same. See https://certification.canonical.com/api/v1/certifiedmakes/?format=json

**Comment by carkod** [comment]
> There are no numbers after the releases. Also the numbers seem to be exactly the same between laptops and desktops - double checking is it just a coincidence?
> image

Fixed. Also removed the path, since it's composed in the template, and removed the `filters=True` from there

**Comment by bethcollins92** [comment]
QA LGTM - not sure if Sheila wants to check the python code?

File: webapp/certification/views.py [hunk 1]
```diff
@@ -195,6 +195,10 @@ def certified_home():
     certified_releases = api.certified_releases(limit="0")["objects"]
     certified_makes = api.certified_makes(limit="0")["objects"]
 
+    # Desktop section
```
Comment (codersquid) [inline webapp/certification/views.py:198]: Update the comment to say Laptops.

**Review by codersquid — COMMENTED** [review]
I have a very minor comment about a comment. see inline comment.

**Review by codersquid — APPROVED** [review]
lgtm

**Pull request merged**